_pages/en/strategy-build-vs-buy.md

```diff
@@ -110,26 +110,27 @@ Stakeholders are expected to still comply with existing policy instruments inclu
 ### Governance, Compliance, and Reporting
 
 1. IT Solutions are broken down into manageable parts (IT Products)
-2. Each IT Product must be assessed on its own as well as a part of overall the IT Solution.
-3. IT Products may be bought, built, or adopted (e.g. Open Source Software)
-4. IT Solutions are assessed against the IT Solutions Risk Assessment Framework (see [Risk Managing the Obtention of Software](#risk-managing-the-obtention-of-software)).
+2. IT Solutions are assessed against the IT Solutions Risk Assessment Framework (see [Risk Managing the Obtention of Software](#risk-managing-the-obtention-of-software)).
   As such:
    1. Complex and core to mandate IT Solutions are architected and delivered internally by ESDC employees but may leverage third party IT Products as components.
    2. Non-complex and not core to mandate IT Solutions may be architected and delivered by third party integrators and/or vendors.
-5. IT Solutions must provide their overall and individual IT products metrics internally to ESDC. Metrics are comprised of the following:
+3. IT Solutions must provide their overall and individual IT products metrics internally to ESDC. Metrics are comprised of the following:
    - Accessibility test results
    - Integration testing results
    - Security test results
    - API consumption statistics
    - Production deployment audit traces
    - Availability statistics
-6. Security Assessment & Authority (SA&A) uses the Target-SA&A methodology
-7. Accessibility Assessment uses the Target-Accessibility methodology
-8. Audit Assessment uses the Target-Audit methodology
-9. IT Solutions reuse existing IT Products, including:
+4. Security Assessment & Authority (SA&A) uses the Target-SA&A methodology
+5. Accessibility Assessment uses the Target-Accessibility methodology
+6. Audit Assessment uses the Target-Audit methodology
+7. IT Solutions reuse existing IT Products, including:
     - Internally developed software
-    - Open Source Software (internally and externally developed)
-    - Third party Services and Products (SaaS, PaaS, and IaaS) and COTS
+    - Open Source Software (internally and externally developed and maintained)
+    - Third party Services and Products (SaaS, PaaS, IaaS, etc.) and COTS
+8. IT Solutions capabilities for which existing IT Products are not available or do not meet the needs can be built internally or contracted for to a third party.
+    - When an IT Product is built internally, or when its custom development is obtained through contracting with a third party, it must be developed in the open and released as open source software by default, in accordance to the ESDC Open Source Framework.
+    - Exceptions may be sought through the Closed-Source Application Development Framework
```

Comment:
Open Source Software includes development tools/frameworks as well, when we develop solutions we are often stitching together open source components. This might be over analytical on my part however, perhaps a definition of open source software is warranted.

Reply:
Definition of Open Source should be at the bottom in the definitions section. 

http://localhost:4000/ITStrategy/strategy-build-vs-buy.html

Reply:
Yes, there's a section for definitions at the bottom but we'll also need to point to existing definitions like the ones provided by TBS: https://www.canada.ca/en/government/system/digital-government/open-source-software/open-first-whitepaper/open-first-whitepaper-use.html#definitions